Content.Server/GameTicking/GameTicker.GamePreset.cs

```diff
@@ -17,6 +17,8 @@ namespace Content.Server.GameTicking
     public sealed partial class GameTicker
     {
         public const float PresetFailedCooldownIncrease = 30f;
+        // What it reverts to if it cannot start the fallback, should probably be a cvar or something
+        private const string SecondaryFallback = "Extended";
```

Comment:
Shouldn't the cvar be an array?

I don't think abraham meant to just hardcode in the secondary.

Reply:
Sadly no. I already tried that with this PR and arrays front work with cvars at all.

Reply:
What's wrong with comma separated

Reply:
What do you mean?

Reply:
Do you mean a string of comma separated values that are then parsed into an array?

Reply:
yeah it would be a bit sus but work

Reply:
It has been done.

Reply:
> yeah it would be a bit sus but work

you're sus.